target/linux/brcm47xx/image/Makefile

```diff
@@ -190,7 +190,9 @@ endef
 ifeq ($(SUBTARGET),generic)
   # BCM4705 with tg3
 define Device/linksys-wrt300n-v1.1
-  DEVICE_TITLE := Linksys WRT300N v1.1
+  DEVICE_VENDOR := Linksys
+  DEVICE_MODEL := WRT300N
+  DEVICE_VARIANT := v1.1
   DEVICE_PACKAGES := kmod-tg3 kmod-b43
   $(Device/linksys)
```

Comment:
You can exploit those includes to provide the DEVICE_VENDOR only once in the parent definition.
This should be possible for asus, linksys, motorola, netgear and maybe others I overlooked.